Netbox installation error

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Netbox

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/netbox.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📈 Which Proxmox version are you on?

8.3.2

### 📝 Provide a clear and concise description of the issue.

I receive an error when installing the Netbox container. After the error the cotainer is running and the webserver is installed but the error looks somewhere in the Netbox install.

### 🔄 Steps to reproduce the issue.

I run the following command:
bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/netbox.sh)"

I tried with advanced settings and also with default settings but both of them give the error. Triefd on multiple proxmox hosts

### ❌ Paste the full error output (if available).

Adding system user `netbox' (UID 105) ...
Adding new group `netbox' (GID 114) ...
Adding new user `netbox' (UID 105) with group `netbox' ...
Not creating `/nonexistent'.
chown: cannot access '/opt/netbox/netbox/media/': No such file or directory

[ERROR] in line 61: exit code 0: while executing command chown --recursive netbox /opt/netbox/netbox/media/

[ERROR] in line 1353: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://git.community-scripts.org/community-scripts/ProxmoxVE/raw/branch/main/install/${var_install}.sh)"

### 🖼️ Additional context (optional).

    _   __     __  ____            
   / | / /__  / /_/ __ )____  _  __
  /  |/ / _ \/ __/ __  / __ \| |/_/
 / /|  /  __/ /_/ /_/ / /_/ />  <  
/_/ |_/\___/\__/_____/\____/_/|_|  
                                   
  ⚙️  Using Default Settings on node 
  🆔  Container ID: 
  🖥️  Operating System: debian (12)
  📦  Container Type: Unprivileged
  💾  Disk Size: 4 GB
  🧠  CPU Cores: 2
  🛠️  RAM Size: 2048 MiB
  🚀  Creating a NetBox LXC using the above default settings
  

  ✔️   Cluster is quorate
  ✔️   Template debian-12-standard_12.7-1_amd64.tar.zst [online]
   💡   Template debian-12-standard_12.7-1_amd64.tar.zst is missing or corrupted. Re-downloading.
  ✔️   Template download successful.
  ✔️   LXC Container 111 was successfully created.
  ✔️   Started LXC Container
  ✔️   Network in LXC is reachable (ping)
  ✔️   Customized LXC Container
  ✔️   Set up Container OS
  ✔️   Network Connected: 192.168.2.48 
  ✔️   IPv4 Internet Connected
   ✖️   IPv6 Internet Not Connected
  ✔️   Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
  ✔️   Updated Container OS
  ✔️   Installed Dependencies
  ✔️   Installed Python
  ✔️   Set up PostgreSQL
⠇ Installing NetBox (Patience)chown: cannot access '/opt/netbox/netbox/media/': No such file or directory

[ERROR] in line 61: exit code 0: while executing command chown --recursive netbox /opt/netbox/netbox/media/

[ERROR] in line 1353: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/install/${var_install}.sh)"

--- a/install/netbox-install.sh
+++ b/install/netbox-install.sh
@@ -56,6 +56,7 @@ RELEASE=$(curl -fsSL https://api.github.com/repos/netbox-community/netbox/releas
 curl -fsSL "https://github.com/netbox-community/netbox/archive/refs/tags/v${RELEASE}.zip" -o "v${RELEASE}.zip"
 $STD unzip "v${RELEASE}.zip"
 mv /opt/netbox-"${RELEASE}"/ /opt/netbox
+mkdir -p /opt/netbox/netbox/media
 
 $STD adduser --system --group netbox
 chown --recursive netbox /opt/netbox/netbox/media/

